redis/client.py

```diff
@@ -3182,7 +3202,9 @@ def _zaggregate(self, command, dest, keys, aggregate=None):
         if aggregate:
             pieces.append(b'AGGREGATE')
             pieces.append(aggregate)
-        return self.execute_command(*pieces)
+        if withscores:
+            pieces.append(b'WITHSCORES')
+        return self.execute_command(*pieces, withscores=withscores)
```

Comment:
@chayim Maybe you will have a better solution for passing the `withscores` property to the options in `execute_cammnd`